src/platform/landing-pages/header-footer-data.json

```diff
@@ -54,7 +54,7 @@
     },
     {
       "column": 2,
-      "href": "https://staging.va.gov/vaforms/",
+      "href": "https://staging.va.gov/find-forms/",
```

Comment:
Interesting, any idea why all the va.gov links here are prefixed with `staging`? Do we have a build task that updates them to prod links in production?

Reply:
I think this file is used for building React apps locally independent of the new content build, so the links are sort of placeholders

Reply:
Ahhh that makes sense 👍